### ISSUE TITLE
Highlight searched value in logs 

### ISSUE THREAD

**Issue by nityanandagohain** [SigNoz/signoz#2366]
When something is searched, it should be highlighted in the logs.

**Comment by nityanandagohain** [comment]
@ankitnayan  to have a scope here.

We are highlighting text in the log body if it's a full-text search. Else if it's a selected field in the filter then the selected field is highlighted in the log ( field won't be highlighted in `raw` view as it is not displayed).

**Comment by ankitnayan** [comment]
That should be fine. The selected fields are removed from the body?

**Comment by nityanandagohain** [comment]
yes depends on the case, ex- `k8s_namespace_name` is not a part of log body.

**Comment by ankitnayan** [comment]
hmm... not keeping `k8s_namespace_name` in the body makes sense? When is a selected field kept in the body then?

**Comment by nityanandagohain** [comment]
any kind of field is present again in the body if the user has not cleaned up the body after extracting the attributes.

But it also depends on the user and the logs that they are printing.


Ex 1:- 
```
{"ts": "1000...", "level":"info", "message":"this is my log"}
```
is parsed to 
```
{
"ts": "1000...",
"body": "this is my log",
"attributes": {
   "level": "info"
  .....
 }
}
```

So in this case the body doesn't contain any attributes and is properly cleaned up.


Ex 1:- 
```
{"ts": "1000...", "level":"asdasd", "message":"this is my log"}
```
is parsed to 
```
{
"ts": "1000...",
"body": "{"ts": "1000...", "level":"info", "message":"this is my log"}",
"attributes": {
   "level": "info"
  .....
 }
}
```
In this case, the body is not cleaned up.

So actually it depends on the user how they are keeping it. But ideally it should be ex 1

**Comment by nityanandagohain** [comment]
Will prioritise this is the next sprint.
